saleor/product/utils.py

```diff
@@ -171,3 +176,11 @@ def price_range_as_dict(price_range):
         return None
     return {'maxPrice': price_as_dict(price_range.max_price),
             'minPrice': price_as_dict(price_range.min_price)}
+
+
+def get_variant_url_from_product(product, attributes):
```

Comment:
Do we need this function? It's used only once in `get_variant_url` below, so maybe instead of introducing it, we could move this logic to `get_variant_url`.